imagetest/fixtures.go

```diff
@@ -79,6 +80,25 @@ func (t *TestWorkflow) LockProject() {
 	t.lockProject = true
 }
 
+// WaitForVMQuota appends a list of quotas to the wait for vm quota step. Quotas with a blank region will be populated with the region corresponding to the workflow zone.
+func (t *TestWorkflow) WaitForVMQuota(qa ...*daisy.QuotaAvailable) error {
+	for _, q := range qa {
```

Comment:
You are only handling this condition because the caller is not taking care of it, if you avoid calling this method if quota is nil then you don't need to handle it here.